bigtop-deploy/juju/hadoop-spark/bundle.yaml

```diff
@@ -29,21 +29,21 @@ services:
       - "2"
       - "3"
   plugin:
-    charm: "cs:xenial/hadoop-plugin-13"
+    charm: "cs:xenial/hadoop-plugin-14"
     annotations:
       gui-x: "1000"
       gui-y: "400"
   client:
     charm: "cs:xenial/hadoop-client-3"
-    constraints: "mem=3G"
+    constraints: "mem=7G root-disk=32G"
```

Comment:
Is this right?  Does the client / gateway node really need such large constraints?

Actually, do we need any application-level constraints if we're already specifying them in the machines section?